src/api/index.ts

```diff
@@ -8,7 +9,7 @@ api.interceptors.request.use((config) => {
   config.headers = {
     ...config.headers,
     "Content-Type": "application/json",
-    Authorization: "Bearer " + localStorage.getItem("token"),
+    Authorization: "Bearer " + localStorage.getItem(TOKEN_KEY),
```

Comment:
Should move get token from local storage outside init function. Should add `Authorization` when we have token, if not, please remove.